roblox/account.py

```diff
@@ -65,3 +65,35 @@ async def set_birthday(
                 "password": password
             }
         )
+    
+    async def get_description(self) -> string:
+        """
+        Gets the authenticated user's description.
+
+        Returns: 
+            The authenticated user's description.
+        """
+        description_response = await self._client.requests.get(
+            url=self._client.url_generator.get_url("accountinformation", "v1/description")
+        )
+        description_data = description_response.json()
+        return description_data["description"]
+    
+    async def set_description(
+            self,
+            description: string,
+    ):
+        """
+        Changes the authenticated user's description.
+        This endpoint *may* require your token, and requires an unlocked PIN.
```

Comment:
This does require a token. I also tend to try to avoid mentioning "this endpoint" in the docs as we're trying to abstract away from endpoints - maybe just say "Updates the authenticated user's description. Requires an unlocked PIN."